### PR TITLE
feat(cdp): show new destinations on the overview page

### PR DESCRIPTION
**Pull request by MarconLP**
## Problem

People have been requesting a slack destination after not finding one on the cdp overview page: https://posthog.slack.com/archives/C077K2Z7UBZ/p1732223332483659
https://posthog.slack.com/archives/C077K2Z7UBZ/p1731818731680879

## Changes
2024-11-22 at 12 58 12@2x

### CODE REVIEW

File: frontend/src/scenes/pipeline/Overview.tsx [hunk 1]
```diff
@@ -46,6 +47,9 @@ export function Overview(): JSX.Element {
                     shown here. <Link to={urls.pipeline(PipelineTab.Destinations)}>See all.</Link>
                 </p>
                 <DestinationsTable />
+                <div className="mt-4" />
+                <h2>New destinations</h2>
+                <DestinationOptionsTable />
```
Comment (benjackwhite) [inline frontend/src/scenes/pipeline/Overview.tsx:52]: I don't like this tbh. That page is already hyper cluttered. Can we just have a button "+ New destination" and link that over to the right page instead?

**Review by benjackwhite — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/pipeline/Overview.tsx (hunk 1 above)
Comment (MarconLP) [reply frontend/src/scenes/pipeline/Overview.tsx]: Changed it to look like this: 2024-11-22 at 14 34 29

File: frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx [hunk 2]
```diff
@@ -8,12 +11,14 @@ export type DestinationsFiltersProps = {
...
     hideShowPaused?: boolean
     hideKind?: boolean
+    askForFeedback?: boolean
 }
 
 export function DestinationsFilters({
     hideSearch,
     hideShowPaused,
     hideKind,
+    askForFeedback = true,
```
Comment (benjackwhite) [inline frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx:21]: Everything else is "hideFoo" - generally speaking when making a change like this it is best to stick with the same naming style please
```suggestion
    hideFeedback,
```

File: frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx [hunk 3]
```diff
@@ -29,9 +34,11 @@ export function DestinationsFilters({
                         onChange={(e) => setFilters({ search: e })}
                     />
                 )}
-                <Link className="text-sm font-semibold" subtle onClick={() => openFeedbackDialog()}>
-                    Can't find what you're looking for?
-                </Link>
+                {askForFeedback ? (
```
Comment (benjackwhite) [inline frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx:37]:
```suggestion
                {!hideFeedback ? (
```

File: frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx [hunk 4]
```diff
@@ -57,6 +64,7 @@ export function DestinationsFilters({
                         onChange={(e) => setFilters({ kind: e ?? null })}
                     />
                 )}
+                {askForFeedback ? null : <NewButton stage={PipelineStage.Destination} size="small" />}
```
Comment (benjackwhite) [inline frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx:67]: Not a fan of this. Why does askForFeedback determine if there is a new button? 

Either this should have its own prop or it should be just done outside of this component.

**Review by benjackwhite — CHANGES_REQUESTED** [review]

File: frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx (hunk 4 above)
Comment (MarconLP) [reply frontend/src/scenes/pipeline/destinations/DestinationsFilters.tsx]: added a `hideAddDestinationButton` property for the new button

**Review by benjackwhite — APPROVED** [review]

**Comment by MarconLP** [comment]
2024-12-02 at 14 00 21

**Pull request merged**